store/keys.go

```diff
@@ -0,0 +1,64 @@
+package store
+
+import (
+	"bytes"
+	"encoding/binary"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// PrefixKeyString returns a byte slice consisting of the prefix string concatenated
+// with all subkeys.
+func PrefixKeyString(prefix string, subkeys ...[]byte) []byte {
+	buf := [][]byte{[]byte(prefix)}
+	return PrefixKeyBytes(append(buf, subkeys...)...)
+}
+
+// PrefixKeyBytes returns a byte slice consisting of all subkeys concatenated together.
+func PrefixKeyBytes(subkeys ...[]byte) []byte {
```

Comment:
How does this differ from `bytes.Join(subkeys, []byte("/"))`?

Reply:
I wasn't aware of the `bytes.Join` API... I'll remove this method.